deployability/modules/allocation/generic/instance.py

```diff
@@ -40,6 +42,8 @@ def __init__(self, path: str | Path, identifier: str, credentials: Credentials =
         self.path: Path = path
         self.identifier: str = str(identifier)
         self.credentials: Credentials = credentials
+        self.host_identifier: Path = Path(host_identifier) if host_identifier else None
+        self.ssh_port: str = ssh_port if ssh_port else None
```

Comment:
In this case you can simply use `or`. Anyway it is probably not required as the default value for `ssh_port` when not received is `None`
```suggestion
        self.ssh_port: str = ssh_port or None
```
```suggestion
        self.ssh_port: str = ssh_port
```